src/components/SingletonEditor.tsx

```diff
@@ -61,20 +61,26 @@ export const useSingletonEditor = (
         return;
       }
 
-      editorState.unlayer.exportHtml((htmlExport: HtmlExport) => {
-        const content = !htmlExport.design
-          ? {
-              htmlContent: htmlExport.html,
-              type: "html",
-            }
-          : {
-              design: htmlExport.design,
-              htmlContent: htmlExport.html,
-              type: "unlayer",
-            };
-
-        onSave(content as Content);
-      });
+      hasChangesRef.current = false;
+      try {
+        editorState.unlayer.exportHtml((htmlExport: HtmlExport) => {
+          const content = !htmlExport.design
+            ? {
+                htmlContent: htmlExport.html,
+                type: "html",
+              }
+            : {
+                design: htmlExport.design,
+                htmlContent: htmlExport.html,
+                type: "unlayer",
+              };
+
+          onSave(content as Content);
+        });
+      } catch (e) {
+        hasChangesRef.current = true;
```

Comment:
I am not sure about this. 

It would catch only limited cases, for example, if `onSave` fails, it will not catch it.

Reply:
You can catch the error from the campaign component.

Reply:
mmm, Ido not understand, how?

Reply:
let's get the try-catch for now

Reply:
Hablamos con @Cromeror, el está pensando en un error handling que vaya más allá, por ejemplo reintentando, etc.

Estoy de acuerdo, pero creo que va más allá del scope de esta PR que se relaciona con un fix del autosave.

Dejamos esos temas para cuando en la práctica nos vayamos encontrando con esos problemas.

Reply:
Hablamos con @Cromeror, el está pensando en un error handling que vaya más allá, por ejemplo reintentando, etc.

Estoy de acuerdo, pero creo que va más allá del scope de esta PR que se relaciona con un fix del autosave.

Dejamos esos temas para cuando en la práctica nos vayamos encontrando con esos problemas.